### ISSUE TITLE
Sound Meter for different board?

### ISSUE THREAD

**Issue by SoLongSidekick** [kirakirakira/pyruler-useful#1]
Sorry to barge into your repo like this, but I just wanted to confirm that the sound meter example is not for the pyruler but for the Circuit Playground, yeah? I assume so but wasn't sure if it might somehow also run fine on the pyruler.

**Comment by kirakirakira** [comment]
@SoLongSidekick You're right! There isn't a speaker on the pyruler 😆 I have a mix of exercises in this repo. Thanks for letting me know I need to clean it up!
